cmd/operator-sdk/internal/genutil/genutil.go

```diff
@@ -47,7 +49,11 @@ func runGoBuildCodegen(binDir, repoDir, genDir string) error {
 	}
 
 	// Only print binary build info if verbosity is explicitly set.
-	if projutil.IsGoVerbose() {
+	if viper.GetBool(flags.VerboseOpt) {
```

Comment:
Instead of checking verbosity here, I'd rather `GOFLAGS="-v"` be set [here](https://github.com/operator-framework/operator-sdk/pull/1361/files#diff-fc95fb53b164f1e3dc64775883195e7bR51), since we call `go` in various places in the SDK. Otherwise looks great!

Reply:
@estroz Hey sorry about my misread there. The "here" link was difficult to interpret but I agree once I figured it out. Please let me know if the error handling there looks appropriate and thanks again for your help here!